cmd/balena/main.go

```diff
@@ -20,7 +20,7 @@ func main() {
 		return
 	}
 	switch filepath.Base(os.Args[0]) {
-	case "balena":
+	case "balenadctl":
```

Comment:
```
balena
balenad
balena-containerd
balena-containerd-shim
balena-containerd-ctr
balena-runc
balena-proxy
```
Out of the original executable names above, I thought that perhaps only `balena` really needed renaming, and it was renamed to `balenadctl` as suggested by Petros. If you think that it is best to rename the other executables as well, please suggest some good names! :-)  By the way:
* I believe that `balena-engined` has already been down-voted by Petros.
* `baleng` would be a nice (?) short name for "balena engine", if it wasn't for the facts that it means "scar" in a Philippine dialect (3rd Google search result) and that baleng.com.au is an Australian company (non-competing line of business, though...).